submodules/wizard/wizard.js

```diff
@@ -1543,11 +1552,13 @@ define(function(require) {
 				technicalContact = accountContacts.technicalContact,
 				controlCenterFeatures = wizardData.creditBalanceAndFeatures.controlCenterAccess.features,
 				accountDocument = {
-					blacklist: _
-						.chain(self.wizardGetStore('apps'))
-						.map('id')
-						.difference(wizardData.appRestrictions.allowedAppIds)
-						.value(),
+					blacklist: (wizardData.appRestrictions.accessLevel === 'full')
```

Comment:
It would be cleaner to declare a variable storing the result of this check. This way, you don't have to conditionally execute the chained methods.

Reply:
Sorry, I did not get the last part. How can be avoided to conditionally execute the chained methods?

```javascript
isFullAccessLevel = wizardData.appRestrictions.accessLevel === 'full',
accountDocument = {
	blacklist: isFullAccessLevel
		? []
		: _
			.chain(self.wizardGetStore('apps'))
			.map('id')
			.difference(wizardData.appRestrictions.allowedAppIds)
			.value(),
```

Reply:
Something like this:
```javascript
apps = wizardData.appRestrictions.accessLevel === 'full'
	? []
	: self.wizardGetStore('apps'),
accountDocument = {
	blacklist: _
		.chain(apps)
		.map('id')
		.difference(wizardData.appRestrictions.allowedAppIds)
		.value(),
```